CompetitionTemplate/src/competition/subsystems/autonomous/selection/BaseAutonomousModeSetCommand.java

```diff
@@ -0,0 +1,21 @@
+package competition.subsystems.autonomous.selection;
+
+import com.google.inject.Inject;
+
+import xbot.common.command.BaseCommand;
+
+public abstract class BaseAutonomousModeSetCommand extends BaseCommand {
+    
+    final AutonomousModeSelector autonomousModeSelector;
+    
+    @Inject
+    public BaseAutonomousModeSetCommand(AutonomousModeSelector autonomousModeSelector) {
+        this.autonomousModeSelector = autonomousModeSelector;
+        this.requires(this.autonomousModeSelector);
+    }
+
+    @Override
+    public void execute() {
+        // no-op, all work done in initialize
```

Comment:
At first I thought that you had forgotten your init method here. =]

Modify the comment to something like: "...all work done in the initialize method of subclasses".